Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -939,6 +1003,8 @@ static bool validateCopyTextureToBuffer(const WGPUImageCopyTexture& source, cons
 
 void CommandEncoder::makeInvalid()
 {
+    [m_commandBuffer commit];
```

Comment:
We need to commit invalid command buffers otherwise we will run out of `-[MTLQueue commandBufferWithDescriptor:`] will block until one is available, which without this commit will never occur.